pylxd/tests/models/test_project.py

```diff
@@ -0,0 +1,166 @@
+import json
+
+from pylxd import exceptions, models
+from pylxd.tests import testing
+
+
+class TestProject(testing.PyLXDTestCase):
+    """Tests for pylxd.models.Project."""
+
+    def test_get(self):
+        """A project is fetched."""
+        name = "test-project"
+        an_project = models.Project.get(self.client, name)
```

Comment:
Please  `s/an_/a_/` here and below (also in text)

Reply:
Done. I don't know why this convention is used in other places. It is odd.
```
$ ag "an_" --py | wc -l
183
```